2020-10-10 00:25:23.434 WARN  --- [Thread-11] com.netflix.discovery.DiscoveryClient Line:1321 - Saw local status change event StatusChangeEvent [timestamp=1602260723434, current=DOWN, previous=UP]
2020-10-10 00:25:27.559 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 00:25:30.746 WARN  --- [Thread-11] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 00:25:31.601 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 00:25:31.601 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient Line:849 - DiscoveryClient_SERVICE/DESKTOP-CTAQJKR:service:8762 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:847)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2020-10-10 00:25:31.601 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.InstanceInfoReplicator Line:125 - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:847)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2020-10-10 00:25:34.835 WARN  --- [Thread-11] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 22:58:09.319 WARN  --- [restartedMain] org.springframework.boot.devtools.autoconfigure.OptionalLiveReloadServer Line:60  - Unable to start LiveReload server
2020-10-10 23:20:43.903 WARN  --- [http-nio-8762-exec-8] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:20:44.887 WARN  --- [http-nio-8762-exec-9] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:20:48.903 WARN  --- [http-nio-8762-exec-1] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:20:48.903 WARN  --- [http-nio-8762-exec-10] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:34.343 WARN  --- [http-nio-8762-exec-3] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:34.343 WARN  --- [http-nio-8762-exec-2] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:36.956 WARN  --- [http-nio-8762-exec-5] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:36.956 WARN  --- [http-nio-8762-exec-4] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:39.357 WARN  --- [http-nio-8762-exec-7] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:39.365 WARN  --- [http-nio-8762-exec-6] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:39.932 WARN  --- [http-nio-8762-exec-11] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:39.932 WARN  --- [http-nio-8762-exec-12] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:41.978 WARN  --- [http-nio-8762-exec-13] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:41.978 WARN  --- [http-nio-8762-exec-14] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:44.357 WARN  --- [http-nio-8762-exec-8] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:44.373 WARN  --- [http-nio-8762-exec-15] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:44.922 WARN  --- [http-nio-8762-exec-9] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:45.178 WARN  --- [http-nio-8762-exec-16] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:49.343 WARN  --- [http-nio-8762-exec-10] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:49.351 WARN  --- [http-nio-8762-exec-1] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:58.549 WARN  --- [http-nio-8762-exec-17] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:22:58.549 WARN  --- [http-nio-8762-exec-18] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:23:03.570 WARN  --- [http-nio-8762-exec-19] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:23:03.578 WARN  --- [http-nio-8762-exec-20] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:25:54.226 WARN  --- [http-nio-8762-exec-17] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:25:54.234 WARN  --- [http-nio-8762-exec-18] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:25:59.242 WARN  --- [http-nio-8762-exec-19] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:25:59.242 WARN  --- [http-nio-8762-exec-20] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:26:00.473 WARN  --- [http-nio-8762-exec-8] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:26:00.489 WARN  --- [http-nio-8762-exec-15] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:26:05.482 WARN  --- [http-nio-8762-exec-9] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:26:05.490 WARN  --- [http-nio-8762-exec-16] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:15.073 WARN  --- [http-nio-8762-exec-10] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:15.073 WARN  --- [http-nio-8762-exec-1] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:20.104 WARN  --- [http-nio-8762-exec-17] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:20.104 WARN  --- [http-nio-8762-exec-18] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:20.229 WARN  --- [http-nio-8762-exec-19] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:20.229 WARN  --- [http-nio-8762-exec-20] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:21.542 WARN  --- [http-nio-8762-exec-8] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:21.542 WARN  --- [http-nio-8762-exec-15] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:25.104 WARN  --- [http-nio-8762-exec-9] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:25.120 WARN  --- [http-nio-8762-exec-16] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:25.229 WARN  --- [http-nio-8762-exec-21] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:25.229 WARN  --- [http-nio-8762-exec-22] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:26.542 WARN  --- [http-nio-8762-exec-23] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:26.542 WARN  --- [http-nio-8762-exec-24] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:30.119 WARN  --- [http-nio-8762-exec-26] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:30.119 WARN  --- [http-nio-8762-exec-25] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:30.323 WARN  --- [http-nio-8762-exec-28] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:30.323 WARN  --- [http-nio-8762-exec-27] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:31.588 WARN  --- [http-nio-8762-exec-29] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:31.588 WARN  --- [http-nio-8762-exec-30] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:35.322 WARN  --- [http-nio-8762-exec-32] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:35.322 WARN  --- [http-nio-8762-exec-31] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:36.588 WARN  --- [http-nio-8762-exec-33] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:29:36.588 WARN  --- [http-nio-8762-exec-34] com.alibaba.druid.pool.DruidDataSource Line:1260 - get connection timeout retry : 1
2020-10-10 23:30:10.663 WARN  --- [Thread-11] com.netflix.discovery.DiscoveryClient Line:1321 - Saw local status change event StatusChangeEvent [timestamp=1602343810663, current=DOWN, previous=UP]
2020-10-10 23:30:37.319 WARN  --- [restartedMain] org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext Line:557 - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aliPayController': Unsatisfied dependency expressed through field 'aliPayService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aliPayServiceImpl': Unsatisfied dependency expressed through field 'alipayOrderMapper'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'alipayOrderMapper' defined in file [E:\works\shopsystem\service\target\classes\com\vtu\mapper\AlipayOrderMapper.class]: Unsatisfied dependency expressed through bean property 'sqlSessionFactory'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'sqlSessionFactory' defined in class path resource [org/mybatis/spring/boot/autoconfigure/MybatisAutoConfiguration.class]: Unsatisfied dependency expressed through method 'sqlSessionFactory' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/alibaba/druid/spring/boot/autoconfigure/DruidDataSourceAutoConfigure.class]: Invocation of init method failed; nested exception is com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2020-10-10 23:36:27.680 WARN  --- [restartedMain] org.springframework.boot.devtools.autoconfigure.OptionalLiveReloadServer Line:60  - Unable to start LiveReload server
2020-10-10 23:39:38.725 WARN  --- [Thread-11] com.netflix.discovery.DiscoveryClient Line:1321 - Saw local status change event StatusChangeEvent [timestamp=1602344378725, current=DOWN, previous=UP]
2020-10-10 23:39:42.748 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 23:39:45.779 WARN  --- [Thread-11] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 23:39:46.763 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
2020-10-10 23:39:46.764 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient Line:849 - DiscoveryClient_SERVICE/DESKTOP-CTAQJKR:service:8762 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:847)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2020-10-10 23:39:46.766 WARN  --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.InstanceInfoReplicator Line:125 - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:847)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2020-10-10 23:39:49.848 WARN  --- [Thread-11] com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient Line:130 - Request execution failed with message: java.net.ConnectException: Connection refused: connect
